flashlight/fl/tensor/backend/jit/opt/Optimizer.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "flashlight/fl/tensor/backend/jit/opt/Optimizer.h"
+
+#include "flashlight/fl/tensor/backend/jit/opt/passes/ScalarFolding.h"
+
+namespace fl {
+
+Optimizer::Optimizer() {
+  passes_.emplace_back(std::make_unique<ScalarFolding>());
```

Comment:
I assume this will be made more configurable later, right? i.e. this isn't a default? 😄

Reply:
Yep. Just haven't figured out the best configuration API yet. Everything in the Optimizer and Evaluator are meant to be minimal but extensible atm.

Dropping a TODO.